vector/v.surf.rst/benchmark/benchmark_v_surf_rst.py

```diff
@@ -0,0 +1,51 @@
+"""Benchmarking of v.surf.rst
+
+@author Aaron Saw Min Sern
+        Chung-Yuan Liang
+"""
+
+from grass.pygrass.modules import Module
+import grass.benchmark as bm
+from subprocess import DEVNULL
+import math
+
+
+def main():
+    results = []
+    # Users can add more or modify existing reference maps
+    raster_cells = [1e6, 2e6, 4e6, 8e6]  # 1M, 2M, 4M, 8M cells
+    for ncells in raster_cells:
+        benchmark(math.sqrt(ncells), f"v.surf.rst_{int(ncells / 1e6)}M", results)
+    bm.nprocs_plot(results)
+
+
+def benchmark(size, label, results):
+    reference = "v_surf_rst_reference_map"
+    output = "benchmark_v_surf_rst_nprocs"
+
+    npoints = size * size * 0.1
+    generate_map(npoints=npoints, rows=size, cols=size, fname=reference)
+
+    module = Module(
+        "v.surf.rst",
+        input=reference,
+        npmin=100,
+        elevation=output,
+        stdout_=DEVNULL,
+        overwrite=True,
+        c=True,
```

Comment:
This flag probably shouldn't be here, no? It doesn't run...
```suggestion
```

Reply:
`c` is a flag to specify running for cross-validation. Without it, it will just run interpolation. It should be kept.